src/app/_utilities/estimateReadTime.ts

```diff
@@ -0,0 +1,7 @@
+export function estimateReadTime(text: string): string {
+  const WPM = 250
+  const wordCount = text.split(/\s+/).length
```

Comment:
Ideally this could be kept on the CMS, through an onUpdate hook or something. Because splitting a string might be costly when the string is an entire blogpost. But it's ok for now 👍

Reply:
noted! will make this an issue